packages/blade/src/components/ListView/types.ts

```diff
@@ -48,8 +48,7 @@ type ListViewFilterProps = {
    */
   selectedFiltersCount?: number;
 } & TestID &
-  DataAnalyticsAttribute &
-  ListViewCommonProps;
+  DataAnalyticsAttribute;
```

Comment:
This won't cause any breaking change right?

Reply:
No `ListViewCommonProps` just had `children` only 1 key.